server/finalise-server.js

```diff
@@ -1,15 +1,11 @@
-var Path = require('path');
-
-module.exports = function finalizeServer(err, server, reactRouted) {
-  if (err) {
```

Comment:
This err stuff no longer serving any functions?

Reply:
If something goes wrong in Hapi now, it'll get thrown, and then caught